tests/test_callbackdatacache.py

```diff
@@ -159,7 +159,8 @@ def test_process_callback_query(self, callback_data_cache, data, message, invali
         if invalid:
             callback_data_cache.clear_callback_data()
 
-        effective_message = Message(message_id=1, date=None, chat=None, reply_markup=out)
+        chat = Chat(1, 'private')
+        effective_message = Message(message_id=1, date=datetime.now(), chat=chat, reply_markup=out)
```

Comment:
is this related to this PR?

Reply:
Ah okay, that's fine then. Just feared that there might be some consequences of the changes that we hadn't seen yet